api-samples/downloads/download_manager/_locales/en/messages.json

```diff
@@ -0,0 +1,162 @@
+{
```

Comment:
@oliverdunk Does the samples page script know to check for messages.json?

Is there a valid reason for internationalizing this sample?

Reply:
The script doesn't check the locale files, I overlooked this. Now I put the name and description into the manifest.json file.